src/wallet/wallet.cpp

```diff
@@ -556,19 +555,26 @@ std::set<uint256> CWallet::GetConflicts(const uint256& txid) const
         return result;
     const CWalletTx& wtx = it->second;
 
+    result = GetConflicts(*(wtx.tx), true);
+
+    return result;
+}
+// tx_is_in_wallet being true means the transaction belongs to wallet and tx_is_in_wallet being false means it is an external transaction
+std::set<uint256> CWallet::GetConflicts(const CTransaction& tx, bool tx_is_in_wallet) const
```

Comment:
In bf9deb407a8d73ac3b645b34c9bd6d28729a809c "rpc/wallet: Add details and duplicate section for simulaterawtransaction"

This refactor of `GetConflicts` could be in a separate commit that precedes this commit.